game.py

```diff
@@ -2,10 +2,11 @@
 from enum import Enum
 from draughts import Board, Move
 import draughts
+import random
```

Comment:
Welcome to this code review! Just so you know, I am going to be extremely nitpicky! :)